src/wp-includes/class-wp-theme.php

```diff
@@ -824,6 +824,38 @@ public function cache_delete() {
 		$this->delete_pattern_cache();
 	}
 
+	/**
+	 * Get block pattern cache.
```

Comment:
Nit-pick: Per WordPress doc standards, we should use third-person verbs.
```suggestion
	 * Gets block pattern cache.
```